xilem/src/style.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//! Traits used to set custom styles on views.
+
+use masonry_winit::core::Property;
+use masonry_winit::properties::types::Gradient;
+use masonry_winit::properties::{
+    Background, BorderColor, BorderWidth, BoxShadow, CornerRadius, Padding,
+};
+use vello::peniko::Color;
+
+/// Trait implemented by views to signal that a given property can be set on them.
+///
+/// You should almost only implement this trait using [`declare_property_tuple!`](crate::declare_property_tuple)
+/// when authoring views.
```

Comment:
```suggestion
/// You should in most cases only implement this trait using [`declare_property_tuple!`](crate::declare_property_tuple)
/// when authoring views.
```
I think "almost only" is ungrammatical.